Dockerfile-Base

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:18.04
+LABEL maintainer = "david.thor@architect.io"
+
+# Install Node.js & Python
+RUN apt-get update && \
+  apt-get -y upgrade && \
+  apt-get install -y build-essential autoconf libtool pkg-config curl git sudo unzip python3-pip && \
+  pip3 install --upgrade pip click grpcio && \
```

Comment:
This pip install and the node install should probably be in different run commands than the apt-get installs so that if one command fails, the others don't need to be re-run for no reason

Reply:
That creates more layers which increases the image size. Recommendations are to perform commands the way shown.